docs/integrations/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Integrations
+sidebar_position: 1
+---
+
+Integrations allow you to import data into Macrometa Global Data Network (GDN) from external data sources or other Macrometa document collections. You can also export data from Macrometa GDN into external data sources or other Macrometa document collections.
+
+Once created, you can reuse integrations as many times as necessary.
+
+## Data Source, Integration, Connection
+
+These terms are used frequently when discussing integrations.
+
+A _data source_ refers to an external database or entity that contains data, such as MongoDB, PostgresSQL, a different Macrometa collection, and so on.
+
+An _integration_ is a connection between Macrometa and a data source. You can create an integration with Macrometa.
```

Comment:
@diana-macrometa - `Integration` is more like a connection configuration to an external entity. 
@dustinlarimer / @elof - Correct me if I'm wrong here.

Reply:
@grainier-macrometa the way we were thinking about it before is that an integration is a group of ways to interact with other platforms and a connection is a type of integration. To add more confusion, I think we are going to move away from calling connectors integrations and just refer to them as connections. @dustinlarimer will be leading whatever changes we need here, but for now, I think the way @diana-macrometa described this will work